view/job_status.go

```diff
@@ -0,0 +1,40 @@
+package view
+
+import (
+	"os/exec"
+
+	"github.com/hashicorp/nomad/api"
+	"github.com/hcjulz/damon/models"
+)
+
+func (v *View) JobStatus(jobID string) {
+	v.Layout.Body.Clear()
+
+	v.Layout.Container.SetInputCapture(v.InputMainCommands)
+	v.Layout.Container.SetFocus(v.components.JobStatus.TextView.Primitive())
+	
+	jobStatus := v.components.JobStatus
+	jobStatus.Status = "Loading..."
+	jobStatus.Render()
+	
+	update := func() {
+
+		cmd := exec.Command("nomad", "status", jobID)
```

Comment:
This is the simplest way to get the status for a job, but the problem I see with this is that it requires the user to have the nomad binary installed on his computer. This doesn't have to be necessariliy the case. For people who don't have the binary installed the `info` command would fail.
Instead what could be done is to get all the information from the nomad API using the go client and put this information together in a text or table view. From the top of my head you should be able to get some of the information by calling this method: https://github.com/hashicorp/damon/blob/main/nomad/jobs.go#L60.

Reply:
Fixed this by using the API instead of shell command and re-using some of the existing functions as well.